src/Player.py

```diff
@@ -18,10 +18,7 @@ def getMovements(self, events):
                 self.keyBools[Player.KEYS.index(event.key)] = True
                 movements[Player.KEYS.index(event.key)] = True
 
```

Comment:
You still need this
```suggestion
            elif event.type == pg.KEYUP and (event.key in Player.KEYS):
                self.keyBools[Player.KEYS.index(event.key)] = False
```
And then you can perform your `or`

Reply:
yes absolutely, I didn't see I deleted that